ppa/editorial/templates/editorial/snippets/attribution.html

```diff
@@ -0,0 +1,10 @@
+{# Snippet to include authors and date attribution to editorial posts #}
+{% for block in page.authors %}
+    {% if block.value.url and add_link %}
+        <a href="{{ block.value.url }}" target="_blank">
+    {% endif %}
+        By {{ block.value.name }}
```

Comment:
This is displaying "By " for every author, not just the first.

Reply:
Will fix and test. Woops.